doc/getting_started.md

```diff
@@ -113,6 +113,8 @@ use the name of your module as the name for your new repository.
     git remote add origin https://github.com/[GITHUB_USER_OR_ORG_NAME]/[GITHUB_MODULE_NAME].git
     git push -u origin master
 
-## 9) Register your method thorugh the Narrative
+## 9) Register your method through the Narrative
+
+Documentation for this is available in the Quick Start section of toplevel SDK documentation.
```

Comment:
"toplevel" -> the top level
How about adding a link to that? https://github.com/kbase/kb_sdk/blob/master/README.md